recipes/molecule/meta.yaml

```
@@ -0,0 +1,75 @@
+{% set name = "molecule" %}
+{% set version = "2.20.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7048ad12bf92f84c2489c9dd2fdc999d72fb344904c75e898ee1742970e0dbad
+
+build:
+  number: 0
```

This looks like it could be `noarch`?
```suggestion
  number: 0
  noarch: python
```

---

It could be, but It so far will not be able to install on windows (due to ansible). If it't doesn't matter if win CI will fail due to dependency then I can change that.